standalone/src/models/product-type/product-type/product-type-draft/presets/sample-data-b2b/heavy-earthmoving-and-construction-equipment.spec.ts

```diff
@@ -224,6 +228,7 @@ describe(`with heavyEarthmovingAndConstructionEquipment preset`, () => {
                 "value": "Capacity",
               },
             ],
+            "level": "Variant",
```

Comment:
@commercetools/craft-team-fe mentioning you here so you are aware of this change.
We're introducing a new property in the `AttributeDefinition` model which has been agreed with the PDM team (it's a beta property they're working on).